File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n=nums.length;
        int i=0;
        for(i=0;i<n-1;i++){
            if(nums[i]==nums[i+1]){
                for(int j=i;j<n-1;j++){
                    nums[j]=nums[j+1];
                }
                n--;
                i--;
                
            }
        }
        return n ;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        map<int,int> m;
        int ans=0;
        for(int i:nums){
            m[i]++;
        }
        for(auto j:m){
            nums[ans++]=j.first;
        }
        return m.size();
    }
};